### DISCUSSION TITLE
exporting project from Activity Browser

### DISCUSSION THREAD

**Discussion by yaramhugo** [LCA-ActivityBrowser/activity-browser#1319] [Q&A] [answered]
Hello:)

How can I export my project in the AB, so i can send it as a file to another person?

Thank you for your help!

Kind regards:)

**Comment by mrvisscher** [comment, marked as answer]
Hi @yaramhugo,

In the newest version of Activity Browser: 2.10.0, you can export your project by clicking on the **File** menu and then selecting **Export this project...**.

If you're not on the latest version you can see how you can update in this discussion: 

- https://github.com/LCA-ActivityBrowser/activity-browser/discussions/1049

Good luck!

Kind regards,

Marin
